.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "src/main/webapp/client"]
+	path = src/main/webapp/client
+	url = https://github.com/terrestris/momo3-frontend
+[submodule "src/main/webapp/lib/BasiGX"]
+	path = src/main/webapp/lib/BasiGX
+	url = https://github.com/terrestris/BasiGX
+[submodule "src/main/webapp/lib/geoext3"]
+	path = src/main/webapp/lib/geoext3
+	url = https://github.com/geoext/geoext3
```

Comment:
`BasiGX` and `GeoExt` submodules will be already used in frontend  --> [[.gitmodules]](https://github.com/terrestris/momo3-frontend/blob/master/.gitmodules)

Maybe we can reuse them?

Note:
if not, it probably would be better to adapt the submodules names at least:
frontend `basigx` and `geoext` against `BasiGX` and `GeoExt3` in backend